### PR TITLE
[TM-FIRST][DNM] Fixes buckshot?

### LINKED ISSUE #6171: Buckshot non-functional
Reporting client version: 513.1538

## Round ID:
2004 (But been around for a very long time)

## Testmerges:
- [[DNM] [TM-ONLY] FLAVOUR TEXT TEMPORARY REVERT](https://github.com/Skyrat-SS13/Skyrat-tg/pull/6116)

## Reproduction:

Get a shotgun and load it with buckshot. Fire. Only one pellet comes out, and it doesn't travel far.

## Runtimes:

 Runtime in projectile.dm, line 691: Cannot read null.x

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Why does putting it in a helper fix it? I don't know. But somehow not doing this broke buckshot. Test merge just in case, but it works fine locally.

Fixes #6171

image

## Why It's Good For The Game

bug fix

## Changelog
:cl:
fix: Shotguns now shoot buckshot properly once more
code: adds ishorrorling() helper for all your horror ling checking needs
/:cl:

### CODE REVIEW

File: code/__DEFINES/is_helpers.dm [hunk 1]
```diff
@@ -158,6 +158,10 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 
 #define isaicamera(A) (istype(A, /mob/camera/ai_eye))
 
+//SKYRAT EDIT ADD START
```
Comment (Funce) [inline code/__DEFINES/is_helpers.dm]: Skyrat "is" helpers go in `code\__DEFINES\~skyrat_defines\is_helpers.dm`

**Review by Funce — COMMENTED** [review]
That's interesting that the helper would adjust the performance so significantly. I'll ask in coderbus.

File: code/__DEFINES/is_helpers.dm (hunk 1 above)
Comment (tf-4) [reply code/__DEFINES/is_helpers.dm]: oops

**Comment by Gandalf2k15** [comment]
I like this regardless.

**Pull request merged**